changelog.md

```diff
@@ -72,6 +76,10 @@ If an application requires IP-per-task, then it can not request ports to be allo
 Currently, this feature does not work in combination with Docker containers. We might still change some
 aspects of the API and we appreciate your feedback.
 
+#### Improved migrations
+It is now possible to resume interrupted ZooKeeper state migrations. That means that Marathon can now recover from
+transient errors during the migration process.
```

Comment:
Fact is it only applies to some specific migrations. I am not sure if it is worth it to clarify this.